lib/index.js

```diff
@@ -2,7 +2,7 @@
 const lowerCase = require('./lower-case')
 const specials = require('./specials')
 
-const regex = /(?:(?:(\s?(?:^|[.\(\)!?;:"-])\s*)(\w))|(\w))(\w*[’']*\w*)/g
+const regex = /(?:(?:(\s?(?:^|[.\(\)!?;:"-])\s*)(\w))|(\w))(\w*[’'àèìòùÀÈÌÒÙáéíóúýÁÉÍÓÚÝâêîôûÂÊÎÔÛãñõÃÑÕäëïöüÿÄËÏÖÜŸçÇßØøÅåÆæœ]*\w*)/g
```

Comment:
Is a different solution for handling these characters than hard-coding this list?